File: Estructura de datos/JavaScript/Recorrido-Filas-Columnas/Filas-columnas.js
// Matriz
let matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

// Recorrido por filas
console.log("Recorrido por filas:");

for (let fila = 0; fila < 3; fila++) {
    for (let columna = 0; columna < 3; columna++) {
        process.stdout.write(matriz[fila][columna] + " ");
    }
    console.log();
}

// Recorrido por columnas
console.log("\nRecorrido por columnas:");

for (let columna = 0; columna < 3; columna++) {
    for (let fila = 0; fila < 3; fila++) {
        process.stdout.write(matriz[fila][columna] + " ");
    }
    console.log();
}

File: Estructura de datos/Python/Recoridos-Filas-Columnas/Filas-columnas.py
# Matriz
TwoDimensionalArray = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

# Recorrido por filas
print("Recorrido por filas:")

for fila in TwoDimensionalArray:
    for elemento in fila:
        print(elemento, end=" ")
    print()

# Recorrido por columnas
print("\nRecorrido por columnas:")

for columna in range(3):
    for fila in range(3):
        print(TwoDimensionalArray[fila][columna], end=" ")
    print()